src/components/text-input/daikin-text-input.ts

```diff
@@ -0,0 +1,184 @@
+import {
+  buttonColorBackgroundPrimaryDisabled,
+  buttonColorBackgroundPrimaryHover,
+  colorFeedbackNegative,
+} from "@daikin-oss/dds-tokens/js/daikin/Light/variables.js";
+import ctl from "@netlify/classnames-template-literals";
+import { LitElement, PropertyValues, css, html, unsafeCSS } from "lit";
+import { customElement, property, state } from "lit/decorators.js";
+import { ifDefined } from "lit/directives/if-defined.js";
+import tailwindStyles from "../../tailwind.css";
+
+export interface DaikinTextInputProps {
+  value: string;
+  type?: "text" | "email" | "tel" | "search";
+  placeholder: string;
+  disabled?: boolean;
+  readonly?: boolean;
+  name?: string;
+  maxlength?: number;
+  autocomplete?: HTMLInputElement["autocomplete"];
+  error?: boolean;
+}
+
+const textInputBase = ctl(`
+  w-[340px]
+  h-12
+  text-daikinNeutral-900
+  border
+  border-solid
+  px-[9px]
+  rounded-[6px]
+  font-daikinSerif
+  placeholder:text-daikinNeutral-200
+
+  enabled:hover:outline
+  enabled:hover:outline-2
+  enabled:hover:outline-[--button-color-background-primary-hover]
+  enabled:active:outline
+  enabled:active:outline-2
+  enabled:active:outline-[--button-color-background-secondary-hover]
+  focus-visible:outline
+  focus-visible:outline-2
+  focus-visible:outline-[--button-color-background-secondary-hover]
+  disabled:text-[--button-color-background-primary-disabled]
+  disabled:bg-[--input-field-color-background]
+  disabled:border-[--button-color-background-primary-disabled]
+`);
+
+const textInputError = ctl(`
+  bg-daikinRed-50
+  border-[--color-feedback-negative]
+`);
+
+/**
+ * Primary UI component for user interaction
+ */
+@customElement("daikin-text-input")
+class DaikinTextInput extends LitElement implements DaikinTextInputProps {
+  static styles = css`
+    ${unsafeCSS(tailwindStyles)}
+
+    :host {
+      --color-feedback-negative: ${unsafeCSS(colorFeedbackNegative)};
+      --button-color-background-primary-hover: ${unsafeCSS(
+        buttonColorBackgroundPrimaryHover
+      )};
+      --button-color-background-primary-disabled: ${unsafeCSS(
+        buttonColorBackgroundPrimaryDisabled
+      )};
+      --button-color-background-secondary-hover: ${unsafeCSS("#CECECE")};
```

Comment:
Confusing name; the design token used in TextInput must be from TextInput. If there is a token from another component with the same value, it is a coincidence.
If the token for TextInput is not yet provided, then it should be treated as a missing token.

Also, there is no need to use unsafeCSS for fixed values.

```suggestion
      --input-text-color-...: #......;
      --input-text-color-...: #......;
      --input-text-color-...: #CECECE;
```